### ISSUE TITLE
 No "Main landmark" is defined for the "Maow" page: A11y_Maow_Home Page_Bypass blocks.

### ISSUE THREAD

**Issue by AnKushSingh05** [microsoft/moaw#113]
### GitHub Tags
#A11yTCS; #A11ySev3; #ChromiumEdge; #WCAG2.4.1; #Win11; #GH_Maow_Web_Mar24; #DesktopWeb; #Maow; #Bypass Blocks; #Benchmark; #A11yMAS;

### Environment Details:
Application Name: Maow
#URL: https://moaw.dev/
OS: Windows 11 Enterprise 23H2 (OS build 22631.3296)
Microsoft Edge: Version 122.0.2365.92 (Official build) (64-bit)

### Repro Steps:

1. Hit the URL https://moaw.dev/
2. Moaw screen will get open.
3. Verify that "Main landmark" is defined for the "Maow" page or not.

### Actual Result:
No "Main landmark" is defined for the "Maow" page.

### Expected Result:
"Main landmark" should be defined for the "Maow" page.
### User Impact:
Screen reader users will not able to navigate the main information easily if the main landmark is not defined for the page.

### Attachment:
![Bug_113](https://github.com/microsoft/moaw/assets/124244478/7d30be76-57e6-4869-bb6d-78817518e29c)

**Comment by sinedied** [comment]
@AnKushSingh05 this one has been fixed, could you validate it?

**Comment by AnKushSingh05** [comment]
Verified the issue in below environment
https://moaw.dev/
issue is fixed. Hence closing the bug.
![Screenshot (1952)](https://github.com/microsoft/moaw/assets/124244478/27cb8c8e-e43b-4616-853b-6f0b451a6504)
